pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -711,9 +715,26 @@ func configureVifV6Addresses(p *MasqueradePodInterface, err error) error {
 }
 
 func (p *MasqueradePodInterface) startDynamicIPServers(vmi *v1.VirtualMachineInstance) error {
+	if err := p.startRADaemon(); err != nil {
+		log.Log.Criticalf("could not start the RA daemon: %v", err)
+		return err
+	}
 	return Handler.StartDHCP(p.vif, p.vif.Gateway, p.bridgeInterfaceName, p.iface.DHCPOptions)
 }
 
+func (p *MasqueradePodInterface) startRADaemon() error {
+	targetPID := "self"
+	if err := Handler.CreateRADaemon(
+		GetNDPConnectionUnixSocketPath(targetPID, p.bridgeInterfaceName),
+		p.bridgeInterfaceName,
+		api.DefaultVMIpv6CIDR,
+		5); err != nil {
```

Comment:
5? Why 5?:) I don't understand why a retry mechanism is needed. But if there is a good explanation, please use a cost for the retries number.

Reply:
This is not required anymore  - ever since I figured out I could specify the selinux context for new sockets, and as a result was able to remove the help process.

Resolving conversation.

Reply:
Can you please explain why with the process the retry mechanism was needed and with the go-routine it isn't?